content/plugins/fabric.md

```diff
@@ -58,6 +63,7 @@ node_templates:
                 - echo "source ~/myfile" >> ~/.bashrc
                 - apt-get install -y python-dev git
                 - pip install my_module
+              use_sudo: true
```

Comment:
perhaps add a comment before the use_sudo: true, specifying that if the property is left out, it is equivalent to passing false